disclosure/settings.py

```diff
@@ -48,10 +53,10 @@
 )
 
 CORS_ORIGIN_ALLOW_ALL = True
-ROOT_URLCONF = 'disclosure_backend.urls'
-WSGI_APPLICATION = 'disclosure_backend.wsgi.application'
+ROOT_URLCONF = 'disclosure.urls'
+WSGI_APPLICATION = 'disclosure.wsgi.application'
 REST_FRAMEWORK = {
-    'VIEW_NAME_FUNCTION': ('disclosure_backend.'
+    'VIEW_NAME_FUNCTION': ('disclosure.'
                            'swagger_nickname.view_name')
```

Comment:
http://django-rest-swagger.readthedocs.org/en/latest/misc.html?highlight=view_name_function#swagger-nickname-attribute

The format itself is the python funciton name. The definition is in `disclosure/swagger_nickname.py` so the module `disclosure.swagger_nickname` contains the function `view_name` so the full "path" is `disclosure.swagger_nickname.view_name` same as you would use if you were `import`ing it.